bitnami/mlflow/templates/tracking/deployment.yaml

```diff
@@ -190,6 +195,16 @@ spec:
             - name: MLFLOW_S3_ENDPOINT_URL
               value: {{ printf "%s://%s:%v" (include "mlflow.v0.s3.protocol" .) (include "mlflow.v0.s3.host" .) (include "mlflow.v0.s3.port" .) | quote }}
             {{- end }}
+            {{- if .Values.externalGCS.enabled }}
+            {{- if .Values.externalGCS.useCredentialsInSecret }}
```

Comment:
I see a gap here, What happens if `externalGCS.enabled=true` and `externalGCS.useCredentialsInSecret=false`? Should users set additional env vars? Should users set additional permissions/roles to the serviceAccount created for mlflow if you are running the chart on GKE cluster?

Reply:
on GCS one common approach is to use Workload Identity. -- What you then do is annotate the serviceaccount with the Google ServiceAccount you'd like to use (this is already supported by the chart), and grant that serviceaccount access to the bucket. It all then "just works" no secrets required.